docs/guide/warnings.rst

```diff
@@ -1475,6 +1475,15 @@ List Of Warnings
    "[WRN:PA0205] No timescale set for "..."".
 
 
+.. option:: UNBASEDUNSIZED
+
+   Warns that unbased unsized literals (:code:`'0`, :code:`'1`, :code:`'z`,
+   :code:`'x`) are SystemVerilog only.
+
+   Ignoring this warning will only suppress the lint check; it will
+   simulate correctly.
```

Comment:
Thanks for the input!

Yosys and Icarus are okay with `'0` even if you specify Verilog-2005, so I think being able to disable it could be a good idea. But now that I've thought about it more, I think it should just be an error. What do you think?

Reply:
Ok, I should have surveyed earlier.  The big 4 all accept this ok with begin_keywords.  With command line flags to disable SV, 2 take it, 1 gives syntax error, and 1 gives a general "new feature" warning somewhat similar to I suggested.

So I think to be fairly compatible we should make it a warning (not error), and go with the generic NEWERSTD idea.